explorer/app/apis/azul/anvil-cmg/common/responses.ts

```diff
@@ -47,7 +48,8 @@ export type DatasetsResponse = AzulHit &
   AggregatedBioSampleResponse &
   AggregatedDonorResponse &
   AggregatedFileResponse &
-  AggregatedLibraryResponse;
+  AggregatedLibraryResponse &
+  AggregatedDiagnosisResponse;
```

Comment:
Diagnosis seemed to be missing from the typing, so I added `AggregatedDiagnosisResponse` to `DatasetsResponse` (since that's the type that's relevant to what I'm doing)

Reply:
Brilliant! Thanks @hunterckx